src/components/BookForm.js

```diff
@@ -0,0 +1,34 @@
+import CATEGORIES from '../utils/categories';
+import './BookForm.css';
+
+const BookForm = () => (
+  <div className="form-div">
+    <form>
+
+      <div className="form-label">
+        <p> Book Title</p>
+        <input id="title" type="text" className="form-control" placeholder="eg Dr, Wahab" />
```

Comment:
- [ ] BookForm component should be placed at the `containers` dir. For now, it's not connected with Redux yet. But it will be in a future milestone.